packages/create-waku/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "module": "ESNext",
+    "moduleResolution": "bundler",
```

Comment:
I don't think this is correct for cli tools. I think it should be `nodenext`, no?

Reply:
bundler is better? Since you are using esbuild here

https://www.typescriptlang.org/tsconfig#moduleResolution

Reply:
For now i think change it to nodenext wont affect anything

Reply:
Oh, I forgot that we use esbuild in cli. Yeah, let's use nodenext if we can. Using esbuild is only a temporary solution anyway.

Reply:
Oh I see a error and its seems correct
<img width="1008" alt="image" src="https://github.com/dai-shi/waku/assets/14026360/4eb76ad1-f607-4ef0-93a9-a58613551206">

Reply:
In this case, create-waku is cjs module but we import a ESM module, I'm not sure the purpose of you're importing this

Reply:
importing fs-extra/esm ?
@vasucp1207 introduced it.
Maybe we should refactor it. My intention is everything should be ESM and no CJS in this repo. (and our plan was to avoid esbuild eventually.)

Reply:
I see, i can do that but it's too far away from this PR

Reply:
That would be great. Feel free to open a new PR.